src/http-client.ts

```diff
@@ -16,21 +16,40 @@ export class HttpRequestError extends Error {
   }
 }
 
-export interface IUniversalFlagConfig {
+export interface IUniversalFlagConfigResponse {
   flags: Record<string, Flag>;
+  bandits: Record<
+    string,
+    {
+      key: string;
+      flagKey: string;
+      variationKey: string;
+      variationValue: string;
+    }
```

Comment:
nit: could extract a `type` here for readability

Reply:
I'm thinking that since this is currently the only place this is used / defined so will leave this as part of the `IUniversalFlagConfigResponse` type to keep it spatially located together, at least until we need to reuse it.